src/components/Booking/container.js

```diff
@@ -55,6 +58,7 @@ export default Container => class extends Component {
         endDate: holiday.endDate || chosenDate,
         halfDay: holiday.halfDay,
         duration: holiday.duration,
+        eventType: holiday.eventType === undefined ? eventType.ANNUAL_LEAVE : holiday.eventType,
```

Comment:
i think here you can just do the following `eventType: holiday.eventType || eventType.ANNUAL_LEAVE,`

Reply:
yes, you're right. i have made the change :)